api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/WorksParams.scala

```diff
@@ -169,17 +168,17 @@ object MultipleWorksParams extends QueryParamsUtils {
           identifiers,
           itemLocationTypeId,
           accessStatus,
-          workType) =>
+          workType,
+          partOf) =>
         // Scala has a max tuple size of 22 so this is nested to get around this limit
         parameter(
           (
-            "partOf".as[PartOfFilter].?,
```

Comment:
Why is this part of this PR?

Reply:
Because of the tuple size restriction thing - I could move it back into the "other" tuple having removed some filters. Annoying!

Reply:
Yea this bit of the params always makes me feel a little uneasy.

Reply:
https://dotty.epfl.ch/docs/reference/dropped-features/limit22.html

Reply:
Looking at this briefly, I wonder if changing the groups would be better?

In particular you get page/pageSize/query/_index as parameters that are common to both MultipleWorkParams and MultipleImageParams. Have a shared function that extracts those (so they're consistent across images/work endpoints), and then you can fit the remaining parameters into a 22-tuple.

It doesn't get you around the problem, exactly, but it gives a slightly clearer distinction of which parameter is unpacked where.

Reply:
I don't massively see the point of coupling certain parameters, but this is my personal thing of keeping things until there is a very clear benefit to do so.